### PR TITLE
Drone: Don't build enterprise Windows installer unless downstream

### PR DESCRIPTION
**Pull request by aknuds1**
**What this PR does / why we need it**:
Drone: Fix enterprise master builds, since they shouldn't try to build the Windows installer unless triggered by OSS.

### CODE REVIEW

File: scripts/lib.star [hunk 1]
```diff
@@ -842,13 +842,10 @@ def get_windows_steps(edition, version_mode, is_downstream=False):
             'rm gcpkey.json',
             'cp C:\\App\\nssm-2.24.zip .',
             '.\\grabpl.exe windows-installer --edition {} --build-id $$env:{}'.format(edition, build_no),
+            '$$fname = ((Get-Childitem grafana*.msi -name) -split "`n")[0]',
+            'gsutil cp $$fname gs://grafana-downloads/{}/{}/'.format(edition, version_mode),
+            'gsutil cp "$$fname.sha256" gs://grafana-downloads/{}/{}/'.format(edition, version_mode),
```
Comment (papagian) [inline scripts/lib.star:847]: out of curiosity: why only in the 2nd command the source file should be quoted?

**Review by papagian — COMMENTED** [review]
Generally LGTM, I have a question

File: scripts/lib.star (hunk 1 above)
Comment (aknuds1) [reply scripts/lib.star:847]: @papagian Good question - it's necessary in this case, to stop PowerShell from interpreting it as getting property `sha256` of `$fname`. Make sense?

Comment (papagian) [reply scripts/lib.star:847]: ah you mean for making the concatenation working; it makes sense

**Review by papagian — APPROVED** [review]

**Pull request merged**